src/u-root.md

```diff
@@ -1,29 +1,19 @@
 # All about u-root
 
-U-root is an embeddable root file system intended to be placed in a flash
-device as part of the firmware image, along with a Linux kernel. The program
-source code is installed in the root file system contained in the firmware
-flash part and compiled on demand. All the u-root utilities, roughly
-corresponding to standard Unix utilities, are written in Go, a modern,
-type-safe language with garbage collection and language-level support for
-concurrency and inter-process communication.
-
-Unlike most embedded root file systems, which consist largely of binaries,
-u-root has only 5: an init program and 4 Go compiler binaries. When a program
-is first run, it, and any not-yet-built packages it uses are compiled to a
-RAM-based file system. The first invocation of a program takes a fraction of a
-second, as it is compiled. Packages are only compiled once, so the slowest
-build is always the first one, on boot, which takes about 3 seconds. Subsequent
-invocations are very fast, usually a millisecond or so.
-
-U-root blurs the line between script-based distros such as Perl Linux[^24] and
+u-root is an embeddable root file system intended to be placed in a flash
```

Comment:
Maybe mention LinuxBoot in this paragraph. That u-root is a LinuxBoot userspace and link to the appropriate generic documentation?

Reply:
done